frontend/src/views/Inventory.LocalBooksList/LocalBooksList.tsx

```diff
@@ -50,6 +58,19 @@ const LocalBooksList: React.FC<Props> = ({
         </>
       ),
     },
+    {
+      title: '',
```

Comment:
El título podría ser `Acciones`.
```suggestion
      title: 'Acciones',
```